<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:#292C34;
color:white;
}</string>
  </property>
  <widget class="QComboBox" name="dropDown">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectLightTypeButton">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>40</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Select Light Type</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="selectLightsRadio">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Lights</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="deselectLightsRadio">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Deselect Lights</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
